### PR TITLE
[4.0] [a11y] System information pages

### PR DESCRIPTION
**Pull request by brianteeman**
Update the pages found at system -> information -> system information to ensure that we have scope for each row and column

### CODE REVIEW

File: administrator/components/com_admin/tmpl/sysinfo/default_phpsettings.php [hunk 1]
```diff
@@ -34,113 +34,113 @@
...
 					<?php echo Text::_('COM_ADMIN_MBSTRING_ENABLED'); ?>
-				</td>
+				</th>
 				<td>
 					<?php echo HTMLHelper::_('phpsetting.set', $this->php_settings['mbstring']); ?>
 				</td>
 			</tr>
 			<tr>
-				<td>
+					<th scope="row">
```
Comment (Quy) [inline administrator/components/com_admin/tmpl/sysinfo/default_phpsettings.php]: Fix indentation

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 4fdddc7883a6e32184b599ac6c58198b5b20f0b1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21186">issues.joomla.org/tracker/joomla-cms/21186</a>.</sub>

**Comment by zwiastunsw** [comment]
What is the reason for these changes? 
These are simple tables. The scope attributes are redundant here. The marking of the header cells in the columns shall be sufficient

**Comment by brianteeman** [comment]
it can always be said to be sufficient but this is more explicit

**Comment by brianteeman** [comment]
Also if we want to encourage best practices so that developers use scope it is simpler if we do it everywhere and not just when someone has made an arbitrary decision that its not a simple table. From experience I can say that if this "simple" table doesnt get the scope update now it wont be long before someone either creates an issue or submits a pull request to "fix" it

**Comment by zwiastunsw** [comment]
I have tested this item :white_check_mark: successfully on 4fdddc7883a6e32184b599ac6c58198b5b20f0b1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21186">issues.joomla.org/tracker/joomla-cms/21186</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21186">issues.joomla.org/tracker/joomla-cms/21186</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**